src/Controller/HomeController.php

```diff
@@ -35,4 +37,25 @@ public function show(ObjectManager $manager, Paginator $paginator, int $page = 1
             'paginator' => $paginator
         ]);
     }
+
+    /**
+     * Get a page of tricks from an ajax call
+     *
+     * @Route("/get-page/{page}", name="home_ajax_get_page", requirements={"page": "\d+"})
+     *
+     * @param ObjectManager $manager
+     * @param Paginator $paginator
+     * @param int $page
+     * @return false|string
+     */
+    public function getPage(ObjectManager $manager, Paginator $paginator, ?int $page = null)
```

Comment:
`int $page = 1` to avoid `$page ?? 1` @line 55?

Reply:
Done.